testing/elasticsearch/APKBUILD

```diff
@@ -0,0 +1,73 @@
+# Contributor: Jakub Jirutka <jakub@jirutka.cz>
+# Maintainer: Jakub Jirutka <jakub@jirutka.cz>
+pkgname=elasticsearch
+pkgver=2.2.1
+pkgrel=0
+pkgdesc="Open Source, Distributed, RESTful Search Engine"
+url="https://www.elastic.co/products/elasticsearch"
+arch="noarch"
+license="ASL 2.0"
+depends="openjdk8-jre"
+makedepends=""
+install="$pkgname.pre-install"
+source="https://download.elasticsearch.org/$pkgname/release/org/$pkgname/distribution/tar/$pkgname/$pkgver/$pkgname-$pkgver.tar.gz
+	$pkgname.initd
+	$pkgname.confd
+	"
+
+_modules="lang-expression lang-groovy"
+for _mod in $_modules; do
+	subpackages="$subpackages $pkgname-$_mod:_${_mod//-/_}"
+	eval "_${_mod//-/_}() { _builtin_module $_mod; }"
+done
+
+_builddir="$srcdir/$pkgname-$pkgver"
```

Comment:
I think it's being set by default as "$builddir" now. @ncopa, could you confirm?